### PR TITLE
Review pages

### PR DESCRIPTION
**Pull request by zc-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by endakelly — APPROVED** [review]

File: source/ways-of-working/new-component/elasticsearch.html.md.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: ElasticSearch
-last_reviewed_on: 2021-10-25
+last_reviewed_on: 2022-05-17
```
Comment (timja) [inline source/ways-of-working/new-component/elasticsearch.html.md.erb:3]: Need @cakeben to review this page I'm not sure if it's still accurate

File: source/ways-of-working/new-component/feature-flags.html.md.erb [hunk 2]
```diff
@@ -21,15 +21,15 @@ Trunk Based Development uses Feature Flags as a mechanism to manage new feature
 
 End to end Process:
 
-1. Developers creates and commit short lived feature branch with Feature flag off.
-2. Impelements features through may iterations of small feature lifecycle.
+1. Developer creates and commits short lived feature branch with Feature flag off.
+2. Impelements feature through iterations of small feature lifecycle.
```
Comment (timja) [inline source/ways-of-working/new-component/feature-flags.html.md.erb:25]:
```suggestion
2. Implements feature through iterations of small feature lifecycle.
```

**Review by timja — COMMENTED** [review]
lgtm, just need ben to check the elastic search page.

(you could also drop the change to it from this PR and merge this now)

**Review by timja — APPROVED** [review]
I've discussed with Ben on Slack, page is accurate although could do with some links and polish. can be done separately though, just need the typo fixed in the above comments

**Pull request merged**